atomic

```diff
@@ -124,6 +124,8 @@ def create_parser(help_text):
                         help=_("show atomic version and exit"))
     parser.add_argument('--debug', default=False, action='store_true',
                         help=_("show debug messages"))
+    parser.add_argument('-i', '--ignore', default=False, action='store_true',
+                        help=_("allows you to ignore certain errors"))
```

Comment:
This is pretty vague.  Either specify what  you are ignoring, or don't give the user the option.